presto-iceberg/pom.xml

```diff
@@ -12,8 +12,8 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <dep.iceberg.version>1.0.0</dep.iceberg.version>
-        <dep.nessie.version>0.43.0</dep.nessie.version>
+        <dep.iceberg.version>1.1.0</dep.iceberg.version>
+        <dep.nessie.version>0.44.0</dep.nessie.version>
```

Comment:
Nessie is being upgraded as part of https://github.com/prestodb/presto/pull/18710